self-hosted/backup-and-restore/docker-and-wale.md

```diff
@@ -9,45 +9,45 @@ tags: [restore, recovery, physical backup]
 
 # Ongoing physical backups with Docker & WAL-E
 
-When you run TimescaleDB in a containerized environment, you can use
+When you run Timescale in a containerized environment, you can use
 [continuous archiving][pg archiving] with a [WAL-E][wale official] container.
 These containers are sometimes referred to as sidecars, because they run
-alongside the main container. We provide a [WAL-E sidecar image][wale image]
-that works with TimescaleDB as well as regular PostgreSQL. In this section, we
-help you set up archiving to your local filesystem with a main TimescaleDB
+alongside the main container. A [WAL-E sidecar image][wale image]
+that works with Timescale as well as regular PostgreSQL. In this section, you 
+can set up archiving to your local filesystem with a main Timescale
 container called `timescaledb`, and a WAL-E sidecar called `wale`. When you are
 ready to implement this in your production deployment, you can adapt the
 instructions here to do archiving against cloud providers such as AWS S3, and
 run it in an orchestration framework such as Kubernetes.
 
 <Highlight type="deprecation">
 This section describes using the WAL-E sidecar for archiving, which is deprecated
-on TimescaleDB. We strongly recommend that you do not use this feature in a
+on Timescale. We strongly recommend that you do not use this feature in a
```

Comment:
There's a partial for this.

Reply:
Thank you @Loquacity !!! just added the partial.